resources/lib/oeWindows.py

```diff
@@ -685,9 +688,9 @@ def onClick(self, controlID):
                             self.is_last_wizard = False
                             break
                 if self.is_last_wizard == True:
-                    xbmc.executebuiltin('UpdateAddonRepos')
```

Comment:
This may be a long shot but it almost looks like the kodi docs are wrong for `executebuiltin` because in the kodi it shows that an extra parameter `wait` can be specified. This should turn it into a blocking call.

I haven't tested it but maybe this would work:
```
xbmc.executebuiltin('UpdateAddonRepos', True)
```

Ref:
Doesn't show it -> https://codedocs.xyz/AlwinEsch/kodi/group__python__xbmc.html#ga2809d4c435f713c6cf4f95c14e926015
Shows it -> https://github.com/xbmc/xbmc/blob/master/xbmc/interfaces/legacy/ModuleXbmc.h#L176

Reply:
Is not working. The parameter is ignored.

```
xbmc.log("Starting")
xbmc.executebuiltin('UpdateAddonRepos', True)
xbmc.log("Started")
```

```
2021-03-06 15:29:20.222 T:1520    DEBUG <general>: Starting
2021-03-06 15:29:20.239 T:1519    DEBUG <general>: CRepositoryUpdateJob[repository.kodi.game] checking for updates.
2021-03-06 15:29:20.239 T:1520    DEBUG <general>: Started
2021-03-06 15:29:20.239 T:1521    DEBUG <general>: CRepositoryUpdateJob[repository.kodinerds] checking for updates.
2021-03-06 15:29:20.239 T:1520     INFO <general>: CPythonInvoker(6, /storage/.kodi/addons/script.hello.world/addon.py): script successfully run
2021-03-06 15:29:20.239 T:1520    DEBUG <general>: onExecutionDone(6, /storage/.kodi/addons/script.hello.world/addon.py)
2021-03-06 15:29:20.240 T:1519    DEBUG <general>: Open - <https://raw.githubusercontent.com/kodi-game/repository.kodi.game/master/release/addons.xml.md5> opening
[...]
```

Reply:
It's not ignored, it's likely that the repo update is run in a separate thread and therefore there is no way to know it has finished 😢

Reply:
you may be able to wait to check for a change to `xbmc.getInfoLabel('Window(AddonBrowser).Property(Updated)')`

see: https://codedocs.xyz/AlwinEsch/kodi/modules__infolabels_boolean_conditions.html#Window_Addonbrowser_Property_Updated

Reply:
Have to perform `xbmc.executebuiltin('ActivateWindow(addonbrowser)')` to read the valid (not empty) property.

Reply:
We don't want to open the AddonBrowser window so the getInfoLabel() call is no solution.

Reply:
I don't think just hoping that the addon repo is updated on time is a great solution either.